atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -473,6 +477,18 @@ def run(self):
             self.upload_scratch_metadata(koji_metadata, server_dir, self.session)
             return
 
+        if self.koji_task_id and (not self.rebuild or (self.rebuild and self.delegate_enabled)):
```

Comment:
can we have comment explaining why?

Reply:
I've actually removed that rebuild check
as all we care about is that we have koji task